scripts/setup-adapters.sh

```diff
@@ -16,7 +16,7 @@
 # Propagate errors and improve debugging.
 set -eufx -o pipefail
 
-SCRIPTDIR=$(dirname "$0")
+SCRIPTDIR=$(dirname "$BASH_SOURCE[0]")
```

Comment:
Got an error:

(error) Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).
See https://github.com/koalaman/shellcheck/wiki/SC1087